components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.jsx

```diff
@@ -116,8 +116,9 @@ export default class PermissionsTree extends React.Component {
     }
 
     updateGroups = () => {
-        const {config, scope} = this.props;
+        const {config, scope, license} = this.props;
         const integrationsGroup = this.groups[this.groups.length - 1];
+        const postsGroup = this.groups[this.groups.length - 2];
```

Comment:
I'm just confused where the order of the groups is defined? Where is it defined that this exact index is where the posts Group is? My worry is just for whatever reason if the order changes, wouldn't these become wrong?

Reply:
yes they would - tests ensure that this order is accurate, it is the order in which the UI renders the permissions

Reply:
Cool! Thanks for verifying.